src/screens/OtherProfile/index.js

```diff
@@ -149,10 +144,9 @@ const OtherProfile = () => {
         setOtherProfileFeed(clonedFeeds, dispatchOtherProfile);
         StorageUtils.otherProfileFeed.setForKey(otherId, JSON.stringify(clonedFeeds));
       }
-      setLoading(false);
       setPostOffset(Number(result.offset));
     } catch (e) {
-      setLoading(false);
+      console.log('error', e);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's good practice to provide more context when logging errors. The current log statement 'console.log('error', e);' could be improved by providing more information about where or why the error occurred.
```diff
- console.log('error', e);
+ console.log('Error while setting post offset:', e);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->